main.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const carousel = document.querySelector('.carousel');
+const nextButton = document.querySelector('.button-next');
+const prevButton = document.querySelector('.button-previous');
+
+nextButton.addEventListener('click', () => {
+  carousel.scrollLeft += 390;
```

Comment:
will be better to not hardcode numbers. Calculate them from imagesCount and imageWidth

But for a simplified example, I think it is ok